transcriptic/cli.py

```diff
@@ -653,6 +658,11 @@ def format_cmd(manifest):
     default=30,
     help="The maximum time in seconds to spend scheduling. The scheduler will use all the time until an optimal solution is found.",
 )
+@click.option(
+    "--schedule-at",
+    default=None,
+    help="The absolute time at which the given protocol should start (at the earliest). Absolute time format YYYY-MM-DDThh:mm in local time (if year, month, or day is missing, it will be auto filled with the current values). Relative time format +NUM in minutes.",
```

Comment:
Could you further clarify `Relative time format +NUM in minutes`? Looking at the code, I understand this as if you specify `+15`, that would imply scheduling 15 minutes in the future. 

IMO we should consider breaking it out into a separate option. (e.g. `schedule-after`)
Its a bit confusing since its a different syntax, and we specified absolute time right at the start of the help message before adding this new format at the end.

Reply:
We should clarify that this time should refer to the scheduling instance's machine time and not the client's machine time (in case of different timezones for example)

Reply:
Using time makes thing complicated!
I will create a different flag for the relative time, seems better. I wanted the option for local testing as it is simpler to do everything in relative if the exact time doesn't matter.

Currently, the absolute time is based on the local time of the client machine. The absolute time is converted to the epoch time for SCLE to deserialize an Instant. Maybe it should be done MCX to make sure that if the user look at the dashboard there is correspondance.